data/856/863/31/85686331.geojson

```diff
@@ -211,10 +211,16 @@
     "wof:breaches":[],
     "wof:concordances":{
         "gp:id":55948203,
+        "iso:code":"MD-TA",
+        "qs:local_id":"30",
```

Comment:
I also saw a `meso:local_id` in another PR.. wondering if this `local_id` should be swapped?

Reply:
Good sleuthing! 

No, they are different sources so I'd like to keep separate. Meso is for county, QS for range of placetypes. So the prefix indicates the source... and both should eventually be swapped for primary sources.

But the official concordance says which to use.